ui/public/scripts/components/modal.directive.js

```diff
@@ -6,9 +6,11 @@
  * @description
  * # modal
  */
-angular.module('mobileControlPanelApp').directive('modal', function($timeout) {
-  return {
-    template: `<button class="btn launch" ng-class={{ngClass}}>{{launch}}</button>
+angular.module('mobileControlPanelApp').directive('modal', [
+  '$timeout',
```

Comment:
Lots of code changes from the formatting but I just added this `$timeout` annotation